### PR TITLE
update the 'Overview' metrics doc for grafana dashboard

### PR DESCRIPTION
**Pull request by bb7133**
### What is changed, added or deleted? (Required)
update `grafana-overview-dashboard.md`

### Which TiDB version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):
TiDB dashboard update: #3284
TiKV dashboard update: #3150

### CODE REVIEW

**Comment by bb7133** [comment]
PTAL @jackysp

File: grafana-overview-dashboard.md [hunk 1]
```diff
@@ -16,61 +16,60 @@ aliases: ['/docs-cn/dev/reference/key-monitoring-metrics/overview-dashboard/']
...
 
 ## TiDB
 
-- Statement OPS：SQL 执行数量统计（包含 select、insert、update 等）
-- Duration：SQL 执行的时间
-- QPS By Instance：每个 TiDB 上的 QPS
-- Failed Query OPM：失败 SQL 的统计，例如语法错误、主键冲突等
+- Statement OPS：不同类型 SQL 每秒执行数量。按 `SELECT`、`INSERT`、`UPDATE` 等来统计
+- Duration：执行的时间
+        - 客户端网络请求发送到 TiDB，到 TiDB 执行结束后返回给客户端的时间。一般情况下，客户端请求都是以 SQL 的形式发送，但也可以包含 COM_PING、COM_SLEEP、COM_STMT_FETCH、COM_SEND_LONG_DATA 之类的命令执行的时间
```
Comment (lilin90) [inline grafana-overview-dashboard.md:41]:
```suggestion
    - 客户端网络请求发送到 TiDB，到 TiDB 执行结束后返回给客户端的时间。一般情况下，客户端请求都是以 SQL 的形式发送，但也可以包含 `COM_PING`、`COM_SLEEP`、`COM_STMT_FETCH`、`COM_SEND_LONG_DATA` 之类的命令执行的时间
```

File: grafana-overview-dashboard.md [hunk 2]
```diff
@@ -16,61 +16,60 @@ aliases: ['/docs-cn/dev/reference/key-monitoring-metrics/overview-dashboard/']
...
+        - 由于 TiDB 支持 Multi-Query，因此，可以接受客户端一次性发送的多条 SQL 语句，如：`select 1; select 1; select 1;`。此时，统计的执行时间是所有 SQL 执行完之后的总时间
```
Comment (lilin90) [inline grafana-overview-dashboard.md:42]:
```suggestion
    - 由于 TiDB 支持 Multi-Query，因此，可以接受客户端一次性发送的多条 SQL 语句，如：`select 1; select 1; select 1;`。此时，统计的执行时间是所有 SQL 执行完之后的总时间
```

File: grafana-overview-dashboard.md [hunk 3]
```diff
@@ -16,61 +16,60 @@ aliases: ['/docs-cn/dev/reference/key-monitoring-metrics/overview-dashboard/']
...
-- Services Offline：各服务 Down 掉节点数量
+- Services Up：各服务在线节点数量
 
 ## PD
 
-- Storage Capacity：TiDB 集群总可用数据库空间大小
-- Current Storage Size：TiDB 集群目前已用数据库空间大小
+- PD role：当前 PD 的角色
+- Storage capacity：TiDB 集群总可用数据库空间大小
+- Current storage size：TiDB 集群目前已用数据库空间大小
```
Comment (kissmydb) [inline grafana-overview-dashboard.md:25]: 是否包括 3 副本

File: grafana-overview-dashboard.md [hunk 4]
```diff
@@ -16,61 +16,60 @@ aliases: ['/docs-cn/dev/reference/key-monitoring-metrics/overview-dashboard/']
...
+- Normal stores：处于正常状态的节点数目
+- Abnormal stores：处于异常状态的节点数目，正常情况应当为 0
 - Number of Regions：当前集群的 Region 总量
```
Comment (kissmydb) [inline grafana-overview-dashboard.md:28]: 是否包括 3 副本

**Comment by bb7133** [comment]
Addressed, PTAL @lilin90 @kissmydb

File: grafana-overview-dashboard.md (hunk 3 above)
Comment (bb7133) [reply grafana-overview-dashboard.md]: Done，PTAL

File: grafana-overview-dashboard.md (hunk 4 above)
Comment (bb7133) [reply grafana-overview-dashboard.md]: Done，PTAL

File: grafana-overview-dashboard.md [hunk 5]
```diff
@@ -6,7 +6,7 @@ aliases: ['/docs-cn/dev/reference/key-monitoring-metrics/overview-dashboard/']
 
 # Overview 面板重要监控指标详解
 
-使用 TiDB Ansible 部署 TiDB 集群时，一键部署监控系统 (Prometheus/Grafana)，监控架构请看 [TiDB 监控框架概述](/tidb-monitoring-framework.md)。
+使用 TiDB Ansible 或 TiUP 部署 TiDB 集群时，一键部署监控系统 (Prometheus/Grafana)，监控架构请看 [TiDB 监控框架概述](/tidb-monitoring-framework.md)。
```
Comment (lilin90) [inline grafana-overview-dashboard.md:9]:
```suggestion
使用 TiDB Ansible 或 TiUP 部署 TiDB 集群时，一键部署监控系统 (Prometheus & Grafana)，监控架构参见 [TiDB 监控框架概述](/tidb-monitoring-framework.md)。
```

File: grafana-overview-dashboard.md [hunk 6]
```diff
@@ -16,61 +16,60 @@ aliases: ['/docs-cn/dev/reference/key-monitoring-metrics/overview-dashboard/']
...
+- Region heartbeat report：TiKV 向 PD 发送的心跳个数
+- 99% Region heartbeat latency：99% 的情况下，心跳的延迟
 
 ## TiDB
 
-- Statement OPS：SQL 执行数量统计（包含 select、insert、update 等）
-- Duration：SQL 执行的时间
-- QPS By Instance：每个 TiDB 上的 QPS
-- Failed Query OPM：失败 SQL 的统计，例如语法错误、主键冲突等
+- Statement OPS：不同类型 SQL 每秒执行数量。按 `SELECT`、`INSERT`、`UPDATE` 等来统计
```
Comment (lilin90) [inline grafana-overview-dashboard.md:39]:
```suggestion
- Statement OPS：不同类型 SQL 语句每秒执行的数量。按 `SELECT`、`INSERT`、`UPDATE` 等来统计
```

File: grafana-overview-dashboard.md [hunk 7]
```diff
@@ -16,61 +16,60 @@ aliases: ['/docs-cn/dev/reference/key-monitoring-metrics/overview-dashboard/']
...
+- Duration：执行的时间
+    - 客户端网络请求发送到 TiDB，到 TiDB 执行结束后返回给客户端的时间。一般情况下，客户端请求都是以 SQL 的形式发送，但也可以包含 `COM_PING`、`COM_SLEEP`、`COM_STMT_FETCH`、`COM_SEND_LONG_DATA` 之类的命令执行的时间
```
Comment (lilin90) [inline grafana-overview-dashboard.md:41]:
```suggestion
    - 客户端网络请求发送到 TiDB，到 TiDB 执行结束后返回给客户端的时间。一般情况下，客户端请求都是以 SQL 语句的形式发送，但也可以包含 `COM_PING`、`COM_SLEEP`、`COM_STMT_FETCH`、`COM_SEND_LONG_DATA` 之类的命令执行的时间
```

**Pull request merged**